appveyor/scripts/sign.ps1

```diff
@@ -0,0 +1,11 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2010-2024 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+param(
+    [string]$ApiToken,
+    [string]$FileToSign
+)
+
+Submit-SigningRequest -ApiToken $ApiToken -InputArtifactPath $FileToSign -OutputArtifactPath $FileToSign -OrganizationId "12147e94-bba9-4fef-b29b-300398e90c5a" -ProjectSlug "NVDA" -SigningPolicySlug "release_signing_policy" -WaitForCompletion -Force
```

Comment:
Should the organizationId be passable instead of hardcoded? Then it could be drawn from a variable like `scons_publisher` is, as @seanbudd pointed out.

Reply:
It could certainly be passable, that isn't too much of an issue. At the moment it's hard coded because I haven't been able to come up with any circumstances under which it would be changed. It would require someone to have an account with SignPath, have their EV/OV hosted by them, and want to sign with their organisation/account. It makes more sense to me that the developer would simply have a locally signed authenticode certificate that they would use.

Reply:
Perhaps it would need to be changed by the NVDA-jp fork? I'm curious as to how they plan to handle the changes to software signing

Reply:
I've reached out to NVDA-jp and they're now aware of this PR. Discussions are happening via email.

If we're concerned about hard-coded values here we'd also need to look at -ProjectSlug "NVDA" and -SigningPolicySlug "release_signing_policy". Both of those are defined inside the SignPath web interface and would be set to different values for different forks / projects. 

My current opinion is that org ID, project, and signing policy are all fine to be hard coded since they're specific to NV Access and won't change. A variable is better for the API key, since it is likely to be rotated and in any case needs to be passed securely. If another fork wants to sign their code using SignPath, they will put their own org/project/policy information in there. But I'm happy for arguments against this approach too.